### DISCUSSION TITLE
How can I improve the speed of "loading a new model"

### DISCUSSION THREAD

**Discussion by Zsural** [lllyasviel/Fooocus#3286] [Q&A] [answered]
I found it's alwags cost much time to load SDXL model, just like below figure.Someone can tell me how to improve it?
![2024-07-15_181642](https://github.com/user-attachments/assets/9c0b045c-2f4f-4bde-8c68-567953522e04)

**Comment by mashb1t** [comment, marked as answer]
Depending on your system specs:
- put Fooocus on an SSD
- have at least 10GB VRAM for fast loading without offloading to RAM / Swap
- have fast RAM

**Reply by Zsural** [reply]
ok.Thank you!

**Comment by bsdesigner** [comment]
After moving the folder from Fooocus to the SSD drive, after running it, I get an error: "PermissionError: [Errno 13] Permission denied:" 
![Знімок екрана 2024-07-15 165445](https://github.com/user-attachments/assets/bd696659-3185-4e9c-a360-0115a26b78a2)

All necessary permissions for the run.bat file have been granted:
![зображення_2024-07-15_170306943](https://github.com/user-attachments/assets/fbcbbd60-c51a-4f8d-ac89-287fba5e99fc)

**Reply by mashb1t** [reply]
@bsdesigner you need to grant permissions for all of the content, in the case of the error it's missing write permissions for the Fooocus folder to write the config modification tutorials txt file. This can be solved by providing write access for all folders and files inside C:\Fooocus\Fooocus_mashb1t (recursively).

**Reply by bsdesigner** [reply]
@mashb1t Thanks, it worked. But I didn't notice any speed increase in loading models after moving to SSD (((

**Reply by mashb1t** [reply]
@bsdesigner multiple factors:
* reading from drive (ssd)
* writing (loading) into VRAM
  * fallback RAM
  * fallback of that swap

Check all of them to prevent bottlenecks.
Make sure you have sufficient VRAM, fast RAM and a fast drive to load from (to VRAM) / write to (swap)
